src/test/init.js

```diff
@@ -26,7 +62,7 @@ test('init()', nest => {
     const mutators = ['a', 'b', 'c'].map(
       (i) => (o, { instance }) => { instance[i] = i; });
 
-    const actual = init(...mutators)();
+    const actual = init(mutators[0], mutators.slice(1, 3))();
```

Comment:
We can have both - `init` in `stamp-utils`, and `init` in `stampit`. They would just do different things.

This module is, as declared, a set of helper functions. No more. Each function receives few arguments and always returns a simplest possible stamp (no added statics, or configuration, or else). So, `stamp-utils` is a minimalistic KISS kind of module. All functions are pure. Like `left-pad`.

Whereas, `stampit` is a full-featured "utility-belt" kind of module. It infects your stamps, adds a zillion statics (including `.init()`), and tries to be "everything you'd every need". Like `lodash`.

Would you agree with such evolution directions?